torch/testing/_creation.py

```diff
@@ -0,0 +1,159 @@
+"""
+This package contains tensor creation utilities.
+"""
+
+import torch
+from typing import Optional, Tuple, Union
+import math
+
+__all__ = [
+    "make_tensor"
+]
+
+def make_tensor(
+    size: Tuple[int, ...],
+    *,
+    device: Union[str, torch.device] = "cpu",
+    dtype: torch.dtype = torch.float32,
+    low: Optional[float] = None,
+    high: Optional[float] = None,
+    requires_grad: bool = False,
+    noncontiguous: bool = False,
+    exclude_zero: bool = False
+) -> torch.Tensor:
+    r"""Creates a random tensor with the given :attr:`size`.
+
+    The function comes with other optional arguments to allow random tensor generation for the user's needs.
+
+    By default, :attr:`device` is ``cpu`` and :attr:`dtype` is ``torch.float32``, but user can pass these arguments
+    as required.
```

Comment:
The docstring should have a separate `Args` section explaining all the parameters. Have a look at how this is done for [`torch.testing.assert_close`](https://github.com/pytorch/pytorch/blob/839eaa2e91556ecd4532596b4fef18a1c3f6e1c1/torch/testing/_asserts.py#L789-L812).